extensions/mssql/src/objectManagement/ui/backupDatabaseDialog.ts

```diff
@@ -476,6 +494,35 @@ export class BackupDatabaseDialog extends ObjectManagementDialogBase<Database, D
 		});
 		return pathMediaMap;
 	}
+
+	private async toggleBackupDestMode(destMode: string): Promise<void> {
```

Comment:
Is Dest means destination here? or for Disk?

Reply:
Got it, back up 'Destinations'...

Reply:
It's a bit confusing, yeah. There's a backup "type", which is like "Full", "Differential", or "Transaction Log". Then there's the "Destination", which is where the backup ends up. So either to Disk or to a storage URL.

Reply:
Renamed some things or better clarity